independent-publisher-connectors/HTTP Garden/apiDefinition.swagger.json

```diff
@@ -0,0 +1,97 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "http garden",
+    "description": "Gardening for every Hyper Text Transfer Protocol response status code. Provides a image for each status code.",

```

Comment:
@troystaylor please make sure that everything is grammatically correct and is free of typographical errors.